buildpack.toml

```
@@ -126,6 +127,10 @@ api = "0.6"
     stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
     uri = "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-21.2.0/graalvm-ce-java8-linux-amd64-21.2.0.tar.gz"
     version = "8.0.302"
+    purl = "pkg:generic/graalvm-jdk@21.2.0"
+    cpes = [
+      "cpe:2.3:a:oracle:graalvm-jdk:21.2.0:*:*:*:*:*:*:*"
```

```suggestion
      "cpe:2.3:a:oracle:graalvm:21.2.0:*:*:*:community:*:*:*",
      "cpe:2.3:a:oracle:jre:1.8.0:update302:*:*:*:*:*:*:*"
```

I'm OK with `graalvm-jdk` for the PURL, but I think we should try to follow the NVD company/product for the CPE.

There is a `oracle:graalvm`. I also see that they use a `enterprise` or `community` designator based on looking at a few of the issues.

To cover the JDK part, I think we should include the Oracle JDK designator as well.